CHANGES

```diff
@@ -73,6 +73,8 @@ Deprecated
 Features added
 --------------
 
+* #3303: Added ``:pyversion:`` option to the doctest directive
```

Comment:
Sorry, this feature will be released as 1.6.
1.5.x series is under maintenance period. so only bug fixes are allowed.